snuba/cli/subscriptions.py

```diff
@@ -166,6 +166,7 @@ def subscriptions(
 
     executor = ThreadPoolExecutor(max_workers=max_query_workers)
     logger.debug("Starting %r with %s workers...", executor, executor._max_workers)
+    metrics.gauge("executor.workers", executor._max_workers)
```

Comment:
I'm actually not entirely sure how this is supposed to be work, to be honest, so I'm just guessing here whether or not this represents what I want it to over time. I suspect this might not be the right thing to do and that I'm reading [the documentation](: https://docs.datadoghq.com/developers/metrics/types/?tab=gauge ) too literally. Their example states that "no new request are received so the value stays the same" for a flush interval. In this case, this gauge would read the correct value (the number of workers doesn't change) — but it seems suspect to me that this behavior would be preserved over a longer time window, or that the value would not overcount after a consumer is shutdown (if the previous value was never evicted.)